server.js

```diff
@@ -19,19 +19,18 @@ app.get('/getPokemon/:name', (req, res) => {
   var name = req.params.name;
   var result = Pokedex.getPokemonByName(name)
     .then(function(response){
-      console.log(`GET /pokemon/${name}`); 
-      res.send(response); 
+      console.log(`GET /pokemon/${name}`);
+      res.send(response);
     })
     .catch(function(){
-      res.send(null); 
-      console.log(`ERROR AT: GET /getPokemon/${name}`); 
-      console.log(`\t Name: ${name}`); 
-    }) 
```

Comment:
Prettier in action, catching weird spaces from my VSCode plugin, without a config file